path/basename.ts

```diff
@@ -20,6 +20,7 @@ import { basename as windowsBasename } from "./windows/basename.ts";
  *
  * @param path - path to extract the name from.
  * @param [suffix] - suffix to remove from extracted name.
```

Comment:
These nits appear throughout the package.
```suggestion
 * @param path Path to extract the name from.
 * @param suffice Suffix to remove from extracted name.
```